.github/scripts/eval_opencompass_config.py

```diff
@@ -552,6 +566,21 @@
                          run_cfg=run_cfg_tp1_template,
                          end_str='[INST]')
 
+# config for llama2-chat-7b-w4a16 turbomind
+tb_llama2_chat_7b_wa416 = dict(
+    type=TurboMindModel,
+    abbr='llama-2-7b-chat-4bits-turbomind',
+    path='meta-llama/Llama-2-7b-chat-hf-inner-4bits',
```

Comment:
do we create the 4bit model manually?

Reply:
> do we create the 4bit model manually?

currently, I use 4bits models produced in daily workflow by default or manually.